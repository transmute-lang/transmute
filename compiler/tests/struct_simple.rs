use common::exec;

mod common;

exec!("struct_simple", "10");
