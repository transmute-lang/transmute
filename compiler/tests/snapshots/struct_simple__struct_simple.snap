---
source: compiler/tests/struct_simple.rs
expression: "format!(\"success:{}\\nstdout:\\n{}\\n\\nstderr:\\n{}\", output.status.success(),\n    String :: from_utf8(output.stdout).unwrap(), String ::\n    from_utf8(output.stderr).unwrap())"
---
success:true
stdout:
f(0) = 1
f(1) = 3
f(2) = 5
f(3) = 7
f(4) = 9
f(5) = 11
f(6) = 13
f(7) = 15
f(8) = 17
f(9) = 19


stderr:
